cdi/src/main/java/org/hibernate/validator/cdi/internal/ValidateableBeanFilter.java

```diff
@@ -42,6 +45,7 @@
  *
  * @author Marko Bekhta
  */
+@IgnoreForbiddenApisErrors(reason = "")
```

Comment:
Hum, why this?

Reply:
let's pretend it was never there :)
was doing some `System.out` and `printStackTrace()` when the Felix test start to fail. But apparently forgot about it when was pushing the changes. Should be ok now.